guides/common/assembly_migrating-from-internal-databases-to-external-databases.adoc

```diff
@@ -4,7 +4,7 @@ ifdef::context[:parent-context: {context}]
 = Migrating from Internal {Project} Databases to External Databases
 
 ifdef::foreman-el,katello[]
-For Red Hat systems only.
+For {RHEL} 7 systems only.
```

Comment:
Technically we can support RHEL 8 (and vanilla Foreman). It's just that we haven't done so yet. Should we open an issue for this?

Reply:
A Redmine? Sure.

Reply:
I think this is 100% a documentation issue. Technically all the code is there, just our instructions are RHEL 7 only.

Reply:
OK so the docs issue is that we need two versions of the docs, one for RHEL 7 and different instructions for RHEL 8? If so I will file an issue.

Reply:
I filed an issue here: https://github.com/theforeman/foreman-documentation/issues/688